distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -11,153 +11,88 @@
 
 == {title}
 
-The Solr Catalog Provider is included with a standard installation of ${branding}.
-The following are the three ways to configure the Solr Catalog Provider:
+The Solr Catalog Provider is included with a standard installation of ${branding}. There are several configurations available:
 
-<<_http_solr,HTTP Solr>> (default):: Used with an external (HTTP) Solr server.
+Internally Managed Solr Server (default):: ${branding} comes bundled with a customized distribution
```

Comment:
I think the double colon is making the text under get tabbed. That may be the desired behavior, just making note.
![image](https://user-images.githubusercontent.com/11355332/41486077-de931c16-7097-11e8-94fb-dc7cdbeca341.png)

Reply:
I _think_ it is desired. That :: was there before so I left it there.